lib/kubernetes-deploy/runner.rb

```diff
@@ -131,6 +138,8 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       success = false
     ensure
       @logger.print_summary(success)
+      tags = %W(namespace:#{@namespace} sha:#{@current_sha})
```

Comment:
This method repeatedly constructs the same tag set. Maybe make a method `statsd_tags` that returns this?